tdms/tests/unit/benchmark_scripts/create_hdf5_test_file.py

```diff
@@ -16,12 +16,11 @@ def create_hdf5_test_file() -> None:
         os.mkdir(os.path.dirname(FNAME_TO_CREATE))
     file = h5py.File(FNAME_TO_CREATE, "w")
 
-    # Create a group under root
-    read_in_test = file.require_group("read_in_test")
-
     # Create test data to read in
     consecutive_numbers = np.arange(0, 12, dtype=float)
 
+    # Create a group under root
+    read_in_test = file.require_group("read_in_test")
```

Comment:
This, however, _does_ need a comment because it's weird to me that `require_group` _creates_ a group.

Reply:
#libhdf5-world-problems

Reply:
That's `h5py` for you :sweat_smile: